spec/acceptance/api/service_spec.rb

```diff
@@ -22,6 +22,9 @@
   end
 
   api 'service' do
+    # stubbing the subscriber to avoid changing the 'updated_at' timestamp on Proxy, and hence on Service
+    before { allow_any_instance_of(ProxyConfigEventSubscriber).to receive(:call) }
```

Comment:
The same issue, and the same solution as for the `Proxy` model: https://github.com/3scale/porta/pull/3960
It turns out, that the `Service`'s `updated_at` is also bumped by the `Proxy`'s touch: